server/services/lduService.js

```diff
@@ -0,0 +1,56 @@
+const logger = require('../../log')
+/**
+ * @typedef {import("../../types/licences").LduService} LduService
+ * @typedef {import("../../types/licences").ActiveLduClient} lduActiveClient
+ * @typedef {import("../../types/delius").DeliusClient} DeliusClient
+ */
+
+/**
+ * @param {DeliusClient} deliusClient
```

Comment:
missing @param for `lduActiveClient`?

Reply:
added. 
Resulted in changes to a number of files because type checking was failing elsewhere